RedHatInsights/ocp-advisor-frontend
# feat(Single rec page): Show risk of change info

Implements https://issues.redhat.com/browse/OCPADVISOR-13.

This enables the risk of change (resolution risk) section at the single recommendation page.

## How to test

Run the PR locally and navigate to /openshift/insights/advisor/recommendations. Select any recommendation and move to the details page. The "Risk of change" header must be rendered with all the additional labels and information. You can also check the API response from `GET /rule/ID` endpoint and check the `resolution_risk` integer value corresponds to the correct label (Very low = 1, Low = 2, Moderate = 3, High = 4).

## Screenshots

![image](https://user-images.githubusercontent.com/31385370/212661511-25f22bb0-b262-4dd4-bcae-f944748616a5.png)
